### PR TITLE
WB Form Field Mobile Adjustments

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/so-widgets-bundle/issues/1087

### CODE REVIEW

File: base/inc/fields/css/date-range-field.less [hunk 1]
```diff
@@ -14,6 +28,9 @@
 
 		input[type="text"].datepicker {
 			width: 258px;
+			@media (max-width: 680px) {
```
Comment (Misplon) [inline base/inc/fields/css/date-range-field.less:31]: Can we perhaps add new line before the media query?

**Comment by Misplon** [comment]
Thanks, mate. Is this PR working for you in the Hero widget? The FitText compressor strength field for example.

**Comment by AlexGStapleton** [comment]
Can you please show me a screenshot of what you're seeing? I just had a look at that field and it looks to be working as expected. It's possible I misinterpreted what you meant by full width and/or this wasn't one of the fields you wanted to be full width.

**Comment by Misplon** [comment]
I have this build running on the demo server so you can check the Corp home
page Hero for example.

**Comment by Misplon** [comment]
Edit_Page_‹_Corp_—_WordPress

**Comment by AlexGStapleton** [comment]
@Misplon Oh. I was looking at a higher resolution so I didn't see that. I can replicate this and I'll fix this now.

**Comment by AlexGStapleton** [comment]
How does that look?

**Pull request merged**